docs/send-data/sumo-logic-distribution-for-opentelemetry-collector/comparison.md

```diff
@@ -0,0 +1,264 @@
+---
+id: comparison
+title: Comparing the Installed Collector and OpenTelemetry Collector
+sidebar_label: OTEL Collector vs. Installed Collector
+---
+
+
+## When to use OpenTelemetry
+
+**Ideal use cases** include when you:
+
+- leverage the **Supported Sources** and **Supported Platforms** listed below
+- are looking for a single agent as opposed to managing multiple agents
+- are having scale issues with FluentD on Kubernetes Collection
+- are looking for ARM support
+
+**Avoid use cases** that:
+
+- are using an **Unsupported Source** or **Unsupported Platform**
+- use CSE due to lack of Windows support
```

Comment:
We now have windows support for otel, I guess this could be updated in the next PR.

Reply:
there's a PR in the opensource docs open for this
https://github.com/SumoLogic/sumologic-otel-collector/pull/1025

Reply:
@rnishtala-sumo @astencel-sumo , thanks! So to confirm, I can delete this line? (`- use CSE due to lack of Windows support`)